tools/tertiary-analysis/scanpy/scanpy-parameter-iterator.xml

```diff
@@ -23,6 +23,7 @@
       <param type="select" name="parameter_name" label="Choose the format of the expression data" help="Use compressed txt, Scanpy or Seurat objects">
```

Comment:
although out of scope, label looks to be a bit weird, right?
```suggestion
      <param type="select" name="parameter_name" label="Parameter name" help="Select a type of parameter">
```

Reply:
I agree it should be relatively useless to have a parameter value... unless it is somehow used for naming stuff down the line.